### ISSUE TITLE
Adding/removing single item fails

### ISSUE THREAD

**Issue by AdamPliska** [viceroypenguin/RBush#13]
I added the following test to reproduce this issue:

The following test fails
```
		[Fact]
		public void VeriftyAddOneRemoveOne()
		{
			var tree = new RBush<Point>();
			var p1 = new Point(1, 1, 1, 1);
			tree.Insert(p1);
			tree.Delete(p1);
		}
```

The following test will pass
```
		[Fact]
		public void VeriftyAddTwoRemoveOne()
		{
			var tree = new RBush<Point>();
			var p1 = new Point(1, 1, 1, 1);
			var p2 = new Point(2, 2, 2, 2);
			tree.Insert(p1);
			tree.Insert(p2);
			tree.Delete(p1);
		}
```

**Comment by DanAvni** [comment]
I had this too. The fix is to change the line below (last line) in the Delete method in RBush.cs
`(path.Peek() as Node).Remove(n);`
Into
`if (!path.IsEmpty) 
    (path.Peek() as Node).Remove(n);`

**Comment by viceroypenguin** [comment]
@DanAvni  Thanks for reminding me of this. I'd totally forgotten about it. 

Can you create a PR for this? I'll pull it in and issue a new release. Thanks.

**Comment by viceroypenguin** [comment]
Released as version 2.0.46
